apache/datadog_checks/apache/apache.py

```diff
@@ -123,12 +123,14 @@ def check(self, instance):
             # Can't get it from the mod_status output, try to get it from the server header even though
             # it may not be exposed with some configurations.
             server_version = r.headers.get("Server")
-            if server_version:
+            if server_version.startswith('Apache'):
```

Comment:
IMHO this check is not needed since the matching pattern is already doing the same (it's somehow a duplication of logic).
```suggestion
            if server_version:
```

and we probably want to get this log if the value is not valid:

```python
self.log.info("Cannot parse the complete Apache version from %s.", value)
```